### DISCUSSION TITLE
Part3. Ch2&3 image 위치 질문드립니다.

### DISCUSSION THREAD

**Discussion by nimo1996** [yoonjeong-kwon/fastcampus-kubernetes#19] [Q&A] [answered]
spec:
  containers:
  - name: oo
    image: 0000 
에서 이미지의 위치가 어디에 있어서 pod로 생성되는지 궁금합니다. 

1. 어느 레포지토리로 연결되어있어서 이미지를 참조하는 것 같은데 이를 확인하는 방법을 모르겠습니다. 
2. 강사님께서 미리 이미지를 구성해두신 곳을 어떻게 참고했는지 궁금합니다.

**Comment by yoonjeong-kwon** [comment, marked as answer]
@nimo1996 님 안녕하세요. 
컨테이너 레지스트리(도커허브)에 컨테이너 이미지를 만들어 업로드 해두었습니다.
name은 실행할 컨테이너 이름을 의미하며
image는 도커 이미지를 다운로드할 위치를 지정합니다.
image를 yoonjeong/my-app:1.0 으로 지정한 경우, 
컨테이너 레파지토리 이름은 yoonjeong이고 컨테이너 이미지는 my-app:1.0 입니다.

https://hub.docker.com/ 에 접속, 로그인 후 검색란에 yoonjeong을 입력해서 확인해보세요. 
도커 관련 구체적인 내용은 패스트캠퍼스 패키지에 포함된 도커 강의를 참고 부탁드립니다.

**Reply by yoonjeong-kwon** [reply]
@nimo1996 추가적으로 도커와 관련된 용어와 개념을 짧게 읽어 보실 수 있도록 마이크로소프트 블로그에 작성된 [Docker 컨테이너, 이미지 및 레지스트리 알아보기](https://learn.microsoft.com/ko-kr/dotnet/architecture/containerized-lifecycle/docker-containers-images-and-registries), [오라클 Docker란 무엇인가?](https://www.oracle.com/kr/cloud/cloud-native/container-registry/what-is-docker/)를 첨부합니다.
